addons/binding/pom.xml

```diff
@@ -1,4 +1,5 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
Please revert the XML changes above this comment.

Can you also add a feature for this binding in `features/openhab-addons/src/main/feature/feature.xml`?
Without it the binding cannot be installed.

Reply:
Good point. And maybe a line for each bundle in the [CODEOWNERS](https://github.com/openhab/openhab2-addons/blob/master/CODEOWNERS) file.

Reply:
pom.xml changed, feature.xml modified.

For the CODEOWNERS? Who do I enter as maintainer?

Reply:
Yourself.

Reply:
Done